sql/core/src/test/resources/sql-tests/inputs/describe-table-after-alter-table.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE table_with_comment (a STRING, b INT, c STRING, d STRING) USING parquet COMMENT 'table_comment';
+
+DESC formatted table_with_comment;
+
+-- ALTER TABLE BY MODIFYING COMMENT
+ALTER TABLE table_with_comment set tblproperties(comment = "modified comment");
```

Comment:
modified the testcase as per comment, added one more proeprty while altering the table.